packages/loader/container-loader/src/test/serializedStateManager.spec.ts

```diff
@@ -48,15 +48,24 @@ const snapshot = {
 	},
 };
 
+const savedOpsSize = 10;
```

Comment:
Given that we now check saved ops at serializedStateManager constructos, I added some here as well to check the expected path